src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -295,7 +296,13 @@ public IEnumerable<PackageItemViewModel> GetCurrent()
                     transitiveToolTipMessage = string.Format(CultureInfo.CurrentCulture, Resources.PackageVersionWithTransitiveOrigins, metadataContextInfo.Identity.Version, string.Join(", ", metadataContextInfo.TransitiveOrigins));
                 }
 
-                ImmutableList<KnownOwnerViewModel> knownOwnerViewModels = LoadKnownOwnerViewModels(metadataContextInfo);
+                ImmutableList<KnownOwnerViewModel> knownOwnerViewModels = null;
```

Comment:
FYI, I think I was the one who encouraged you to use `ImmutableList` in the first place. This was unintentionally bad advice, because it's a linked list. There's `ImmutableArray<T>` which is what I should have recommended if I knew the difference at the time.

But it's something that can be done later.